src/components/nav/nav.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+nav {
+  display: flex;
+  width: 160px;
+  height: 1000px;
+  color: #fff;
+  background-color: RGBA(33,37,41,var(--bs-bg-opacity,1));
```

Comment:
Could you please change this background-color style to HEX type as "color: #fff"?